hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/expiry/removal/TreasuryReturns.java

```diff
@@ -20,7 +20,15 @@
 import static com.hedera.services.state.expiry.removal.FungibleTreasuryReturns.UNFINISHED_NOOP_FUNGIBLE_RETURNS;
 import static com.hedera.services.state.expiry.removal.NonFungibleTreasuryReturns.FINISHED_NOOP_NON_FUNGIBLE_RETURNS;
 import static com.hedera.services.state.expiry.removal.NonFungibleTreasuryReturns.UNFINISHED_NOOP_NON_FUNGIBLE_RETURNS;
-import static com.hedera.services.throttling.MapAccessType.*;
+import static com.hedera.services.throttling.MapAccessType.ACCOUNTS_GET;
```

Comment:
Did `spotlessApply` get back these ? Or is it Intellij setting ?

Reply:
I have configured IntelliJ to not use * imports